jmdaemon/test/test_daemon_protocol.py

```diff
@@ -66,7 +66,8 @@ def clientStart(self):
                             network="dummynetwork",
                             irc_configs=json.dumps(irc),
                             minmakers=2,
-                            maker_timeout_sec=3)
+                            maker_timeout_sec=3,
+                            dust_threshold=27300)
```

Comment:
Of course, it's just a test, but why not use 2730 also here, for consistency?

Reply:
I agree it doesn't matter, but this one should be 27300 no? It's `jm_single().DUST_THRESHOLD` being passed over from client.